contracts/misc/NFTFloorOracle.sol

```diff
@@ -162,12 +162,20 @@ contract NFTFloorOracle is Initializable, AccessControl, INFTFloorOracle {
         _addFeeders(_feeders);
     }
 
-    /// @notice Allows owner to remove feeder.
-    /// @param _feeder feeder to remove
-    function removeFeeder(address _feeder)
+    /// @notice Allows owner to remove feeders.
+    /// @param _feeders feeders to remove
+    function removeFeeders(address[] calldata _feeders)
         external
-        onlyWhenFeederExisted(_feeder)
+        onlyRole(DEFAULT_ADMIN_ROLE)
     {
+        for (uint256 i = 0; i < _feeders.length; i++) {
+            _removeFeeder(_feeders[i]);
+        }
+    }
+
+    /// @notice Allows owner to remove feeder.
+    /// @param _feeder feeder to remove
+    function removeFeeder(address _feeder) external {
```

Comment:
This function need access control. or we can remove it since we have `removeFeeders` already.